public/pages/RollupDetails/containers/RollupDetails/RollupDetails.tsx

```diff
@@ -125,12 +125,10 @@ export class RollupDetails extends Component<RollupDetailsProps, RollupDetailsSt
         ? [BREADCRUMBS.ROLLUPS, { text: id }]
         : [BREADCRUMBS.INDEX_MANAGEMENT, BREADCRUMBS.ROLLUPS, { text: id }];
       this.context.chrome.setBreadcrumbs(newBreadCrumbs);
-      this.props.history.push(`${ROUTES.ROLLUP_DETAILS}?id=${id}`);
```

Comment:
Let's please make sure we are not regressing on anything. Please make sure if old UI is working as expected.

Reply:
Checked